deploy-as-code/helm/charts/digit-works/backend/contracts/values.yaml

```diff
@@ -109,6 +109,11 @@ env: |
       configMapKeyRef:
         name: egov-service-host
         key: egov-location
+  - name: WORKS_MUSTER_ROLL_HOST
```

Comment:
We should not be tightly coupling contract service with muster roll. @jagankumar-egov - we are getting more and more services tightly coupled with the contract service. We need a BFF pattern for this.

Reply:
This needs to be addressed in a later release as technical debt. Had a discussion with @shubhang-eGov and @jagankumar-egov about this. UI team to come up with a BFF proposal.